package.json

```diff
@@ -41,7 +41,8 @@
   },
   "scripts": {
     "lint": "standard && node ./lib/rtlcss.js",
```

Comment:
ideally this should be split too so one can do `npm run lint -- --fix`

Reply:
Would you update the PR ? and Thanks for contributing!

Reply:
How do you want to name the new `node ./lib/rtlcss.js` script? Also, do you want me to add a `lint:fix` script?

Reply:
I was thinking 😅 

```json
"lint": "standard && node ./lib/rtlcss.js",
"lint:fix": "standard --fix && node ./lib/rtlcss.js",
```

Reply:
The thing with this approach is that doing `npm run lint -- --fix` won't work as expected. So, ideally, the `node ./lib/rtlcss.js` should be moved to a separate one. But it's your call. Personally I sort of expect this behavior.

Reply:
Both are fine with me as long as I don't need to come up with the name 😇